setup.cfg

```diff
@@ -1,49 +1,92 @@
 [metadata]
 name = sunkit_pyvista
-author =
-author_email =
+provides = sunkit_pyvista
+description = 3D visualisation of solar physics data with pyvista
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = The SunPy Community
+author_email = sunpy@googlegroups.com
 license = BSD 3-Clause
 license_file = licenses/LICENSE.rst
 url = https://github.com/sunpy/sunkit-pyvista
-description = 3D visualisation of solar physics data with pyvista
-long_description = file: README.rst
+edit_on_github = True
+github_project = sunpy/sunkit-pyvista
+platform = any
+keywords = solar physics, solar, science, sun, 3d, plotting, visualisation
+classifiers =
+  Development Status :: 3 - Alpha
+  Intended Audience :: Science/Research
+  License :: OSI Approved :: BSD License
+  Natural Language :: English
+  Operating System :: OS Independent
+  Programming Language :: Python
+  Programming Language :: Python :: 3
+  Programming Language :: Python :: 3.7
+  Programming Language :: Python :: 3.8
+  Programming Language :: Python :: 3.9
+  Topic :: Scientific/Engineering :: Visualization
 
 [options]
+python_requires = >=3.7
 zip_safe = False
 packages = find:
-python_requires = >=3.7
-setup_requires = setuptools_scm
+include_package_data = True
+setup_requires =
+  setuptools_scm
 install_requires =
-    sunpy>=3
-    pyvista
-
-
-[options.entry_points]
-console_scripts =
-    astropy-package-template-example = packagename.example_mod:main
-
+  pyvista>=0.30.1
+  sunpy>=3.0.0
 
 [options.extras_require]
-all =
-test =
+tests =
     pytest
-    pytest-doctestplus
     pytest-cov
+    pytest-doctestplus
 docs =
     sphinx
     sphinx-automodapi
-    sunpy-sphinx-theme
     sphinx-automodapi
+    sunpy-sphinx-theme
 
 [options.package_data]
 sunkit_pyvista = data/*
 
 [tool:pytest]
 testpaths = "sunkit_pyvista" "docs"
+norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" ".history"
 doctest_plus = enabled
+doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
 text_file_format = rst
 addopts = --doctest-rst
 
+[pycodestyle]
+max_line_length = 110
+
+[flake8]
+max-line-length = 110
+exclude =
+    .git,
+    __pycache__,
+    docs/conf.py,
+    build,
+    sunkit_pyvista/__init__.py,
+rst-directives =
+    plot
+
+[isort]
+balanced_wrapping = True
+skip=docs/conf.py,sunkit_pyvista/__init__.py
+default_section = THIRDPARTY
+include_trailing_comma = True
+known_astropy = astropy, asdf, sunpy
```

Comment:
A decision to make, do we want to just dump astropy and sunpy together in the imports or just seperate them?